less/theme.less

```diff
@@ -124,3 +124,8 @@
 {
   display: none;
 }
+
+.div-inline-block
```

Comment:
I don't really like this class name. I said I like specific class names as in the class name should say what the element is and then all the styling for it can be in CSS. This is opposed to an element having five classes like "inline-block centered offset-left other-modifier". I feel that this keeps the dom cleaner. But I don't really care anymore. This name is a little silly though "div" is redundant. So just call it inline-block, checkbox, or create-league-checkbox